Libraries/libs.mk

```diff
@@ -206,9 +206,52 @@ ifeq ($(LIB_EMV), 1)
 EMV_DIR ?= $(LIBS_DIR)/EMV
 
 ifeq ("$(wildcard $(EMV_DIR))","")
-$(error EMV library not found. Please install the EMV package to $(EMV_DIR))
+$(error EMV library not found (Only available via NDA). Please install the EMV package to $(EMV_DIR))
 endif
 
 include $(EMV_DIR)/emv.mk
 endif
 # ************************
+
+# UCL (Disabled by default)
+# Only available via NDA
+# ************************
+LIB_UCL ?= 0
+ifeq ($(LIB_UCL), 1)
+
+TARGET_SEC ?= $(TARGET_UC)
+TARGET_SEC_LC := $(subst M,m,$(subst A,a,$(subst X,x,$(TARGET_SEC))))
+
+PROJ_CFLAGS += -DWORD32 -DM_SDK
+
+ifeq ($(TARGET_SEC), MAX32520)
```

Comment:
@ozersa What do you think about moving `TARGET_SEC_LC` calculation and this section into its own Libraries/UCL/UCL.mk file?

Inputs to UCL.mk:
* `TARGET_SEC`

Outputs from UCL.mk:
* `PROJ_CFLAGS += ...`
* `IPATH += $(UCL_DIR)/$(TARGET_UC)/include`
* `PROJ_LDFLAGS += -L$(UCL_DIR)/$(TARGET_UC)`
* `PROJ_LIBS += ucl_$(UCL_VERSION)_$(TARGET_SEC_LC)`

I would prefer if the target-specific calculations are done in a separate file, similar to what we did for [SBT-config.mk](https://github.com/Analog-Devices-MSDK/msdk/blob/main/Tools/SBT/SBT-config.mk)

Reply:
Done

Reply:
Thanks, looks good!